applications/nrf5340_audio/src/audio/Kconfig

```diff
@@ -192,7 +192,11 @@ config LC3_BITRATE_MIN
 config LC3_BITRATE
 	int
 	range LC3_BITRATE_MIN LC3_BITRATE_MAX
-	default BT_AUDIO_BITRATE
+	default BT_AUDIO_BITRATE_SINK
+
+config LC3_BITRATE_RETURN
+	int
+	default BT_AUDIO_BITRATE_SOURCE
```

Comment:
General: Add OCT-XXXX number to the commit message.

Reply:
Done